### PR TITLE
Support Cocoapods & Carthage

### PR DESCRIPTION
**Pull request by biscuitehh**
Fixes issue #6

### CODE REVIEW

File: Example/Pods/Local Podspecs/WillowTreeReachability.podspec.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "MIT",
+    "file": "LICENSE"
+  },
+  "authors": "WillowTree, Inc.",
+  "platforms": {
+    "ios": "8.0"
+  },
+  "source": {
+    "git": "https://github.com/willowtreeapps/WillowTreeReachability.git",
+    "tag": "2.0.0"
```
Comment (eriklamanna) [inline Example/Pods/Local Podspecs/WillowTreeReachability.podspec.json]: You'll need to update this tag as well

File: Example/Pods/Target Support Files/Pods-ReachabilityExample/Pods-ReachabilityExample-acknowledgements.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Acknowledgements
+This application makes use of the following third party libraries:
+
+## WillowTreeReachability
+
+Copyright (c) 2015 WillowTree, Inc.
```
Comment (eriklamanna) [inline Example/Pods/Target Support Files/Pods-ReachabilityExample/Pods-ReachabilityExample-acknowledgements.markdown]: We should probably bump the app copyright to 2016

File: Example/Pods/Local Podspecs/WillowTreeReachability.podspec.json (hunk 1 above)
Comment (biscuitehh) [inline Example/Pods/Local Podspecs/WillowTreeReachability.podspec.json]: Yeah - it's an auto generated Pod file but I saw that after the fact, good catch.

**Comment by biscuitehh** [comment]
It looks like Xcode 8.1 uses Swift 3.0.1 which is causing the error on Travis/Xcode 8. Silly Xcode.

**Pull request merged**